community-containers/jellyseerr/readme.md

```diff
@@ -0,0 +1,16 @@
+## Jellyseerr
+This container bundles Jellyseerr and auto-configures it for you.
+
+### Notes
+- After adding and starting the container, you can directly visit http://ip.address.of.server:5055 / and access your new Jellyseerr instance, which can be used to manage Plex, Jellyfin, and Emby.
+- In order to access your Jellyseerr outside the local network, you have to set up your own reverse proxy. You can set up a reverse proxy following [these instructions](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md) and [Jellyseerr's reverse proxy documentation.](https://docs.jellyseerr.dev/extending-jellyseerr/reverse-proxy)
+- If you have a firewall like ufw configured, you might need to open port 5055 so that you can access the webui from you local network.
```

Comment:
the config in the json overwrites any ufw setting. So this should not be needed.

```suggestion
```